typed-racket-more/typed/file/glob.rkt

```diff
@@ -0,0 +1,16 @@
+#lang typed/racket/base
+
+(require typed/racket/unsafe)
+
+(define-type Glob (U Path-String (Sequenceof Path-String)))
+
+(unsafe-require/typed file/glob
+  [glob (->* [Glob] [#:capture-dotfiles? Boolean] (Listof Path-String))]
+  [in-glob (->* [Glob] [#:capture-dotfiles? Boolean] (Sequenceof Path-String))]
+  [glob-match? (->* [Glob Path-String] [#:capture-dotfiles? Boolean] Boolean)]
+  [glob-capture-dotfiles? (Parameterof Boolean)])
+
+(unsafe-provide glob
```

Comment:
We definitely don't need both `unsafe-require` and `unsafe-provide`. Presumably no one will use this library from untyped code.